packages/addons/addon-depends/python/version_control/python/setup.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+from setuptools import setup
+
+setup(name='version_control',
+      version='1.0',
+      description='A version controlled bundle of Python packages',
+      author='Team LibreELEC',
+      url=' https://libreelec.tv',
+      py_modules = ['dummy'],
+      install_requires=[
+          "cryptography>=1.9",
```

Comment:
But this one should be "==" otherwise it would install 2.0 if it were available?